app/frontend/styles/components/schools-in-your-trust.scss

```diff
@@ -0,0 +1,17 @@
+.missing-information {
+  border-top: 5px solid $govuk-brand-colour;
+  padding-top: govuk-spacing(2);
+}
+
+.schools {
+  border-top: 5px solid govuk-colour("orange");
+  margin-top: govuk-spacing(2);
+  padding: govuk-spacing(2) 0;
+}
+
+.school-table {
+  border: 10px solid govuk-colour("mid-grey");
```

Comment:
This should be `$govuk-border-colour`

Reply:
Even though they're the same thing technically, so it doesn't really matter

Reply:
I do prefer `$govuk-border-colour` thanks, will change it